### PR TITLE
[Based on #688] Add a screen shows contributors

### LINKED ISSUE #69: Add contributors screen
## Overview (Required)
- We need to create the contributors screen

## Links
- https://gallery.io/projects/MCHbtQVoQ2HCZd9Pbpex4PVP/files/MCHtA7U1iMGr64hSKBFaITmFHhU5Yn4CQP8 (Please send request)

### PR DESCRIPTION
**Pull request by jmatsu**
## Issue
- close #69

## Overview (Required)
- I've taken over the task to add a contributor screen from #688 
- Fixed several bugs, typos and introduced assisted dagger to open urls from binding items.

## Links
- N/A

## Screenshot
Before | After
:--: | :--:
N/A |

### CODE REVIEW

**Comment by jmatsu** [comment]
Yay, completed. I FOUND A BUG ON THE API SIDE. 🎉 😞

**Comment by jmatsu** [comment]
@takahirom could you please have a look? 🙏

File: data/api-impl/src/commonMain/kotlin/io/github/droidkaigi/confsched2019/data/api/response/ContributorItemReesponseImpl.kt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package io.github.droidkaigi.confsched2019.data.api.response
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class ContributorItemReesponseImpl(
```
[the remark this answers is not in the document]
Comment (jmatsu) [reply data/api-impl/src/commonMain/kotlin/io/github/droidkaigi/confsched2019/data/api/response/ContributorItemReesponseImpl.kt]: Grep expert...

**Comment by takahirom** [comment]
LGTM 👍

**Pull request merged**